src/Microsoft.IdentityModel.Clients.ActiveDirectory/Internal/Instance/Authenticator.cs

```diff
@@ -122,10 +125,35 @@ public async Task UpdateFromTemplateAsync(RequestContext requestContext)
                 {
                     ServiceBundle.InstanceDiscovery.AddMetadataEntry(host);
                 }
-                AuthorizationUri = ServiceBundle.InstanceDiscovery.FormatAuthorizeEndpoint(host, tenant);
-                DeviceCodeUri = string.Format(CultureInfo.InvariantCulture, "https://{0}/{1}/oauth2/devicecode", host, tenant);
-                TokenUri = string.Format(CultureInfo.InvariantCulture, "https://{0}/{1}/oauth2/token", host, tenant);
-                UserRealmUriPrefix = EnsureUrlEndsWithForwardSlash(string.Format(CultureInfo.InvariantCulture, "https://{0}/common/userrealm", host));
+
+                //Check if custom port is set on authority
+                if (authorityUri.Port != 443)
```

Comment:
Did you mean to reference "DefaultHttpsPort" const declared above?

Reply:
Yes. I forgot to push my last commit. Thanks